apps/tlon-web/src/state/channel/channel.ts

```diff
@@ -1165,6 +1461,25 @@ export function useSortMode(nest: string): SortMode {
   return channel?.sort ?? 'time';
 }
 
+const emptyPending: PendingMessages = {
+  posts: {},
+  replies: {},
+};
+export function usePendingPosts(nest: Nest) {
+  const channel = useChannel(nest);
+  return channel?.pending || emptyPending;
+}
+
+export function useIsSentPost(nest: Nest, cacheId: CacheId) {
```

Comment:
may be good to keep these around, but doesn't seem like `useIsSentPost` or  `useIsSentReply` are used?